features/networking/pod.feature

```diff
@@ -525,3 +525,80 @@ Feature: Pod related networking scenarios
       | ls /var/lib/cni/networks/openshift-sdn/ |
     Then the step should succeed
     And the output should not contain "10.132.2.200"
+
+  # @author zzhao@redhat.com
+  # @case_id OCP-41666
+  # @bug_id 1924741
+  @admin
+  @destructive
+  Scenario: Pod stuck in container creating - failed to run CNI IPAM ADD: failed to allocate for range 0: no IP addresses available in range set
+    Given I switch to cluster admin pseudo user
+    When I run the :label admin command with:
+      | resource  | machineconfigpool         |
+      | name      | worker                    |
+      | key_val   | custom-kubelet=large-pods |
+      | overwrite | true                      |
```

Comment:
Do we restore the change here ?

Reply:
I think this should not be harmful for the other testing.  I will also delete this step `Given admin ensures "set-max-pods" kubelet_config is deleted after scenario`, thus can save a lot of time to restore the cluster. 
@liangxia WDYT?

Reply:
since this restore will reboot all worker and will cost about 10 mins.  this kubeconfig should not be affected others.

Reply:
@pruan-rht @jhou1 How do you think ?

Reply:
@zhaozhanqi Did you mean by deleting the `set-max-pods` KubeletConfig, the removal of this label is also ensured?

Reply:
@jhou1 sorry for missing your comment.  
Yes, I mean I'm considering if we do not need to delete the set-max-pods KubeletConfig after the scenario since this will cost a long time and  IMO, this should not be affect other test. WDYT?

Reply:
@zhaozhanqi IMO, if you are certain having this `set-max-pods` kubeletconfig does not interfer with other tests and deleting it would cost at least 10 minutes (more if there are more workers in the cluster), then I would vote to keep it.  The drawback is that if in the future this setting cause problems for other tests then it's hard to pin-point it.

Reply:
@pruan-rht  this config for `set-max-pods` is common setting and it just expand the pods number of node . so in theory, it should not be caused any problems in the cluster and it maybe a bug if other testing failed duo this updated.

Reply:
@zhaozhanqi ok, sound reasonable to not clean it up then to save over 10 minutes of idle wait.

Reply:
can you help merge if it's ok? thanks